infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -751,8 +750,6 @@ private void onStandbyClusterAddRemove(TopologyDescriptor discoveredTopology) {
         updateLocalTopology(discoveredTopology);
         updateReplicationManagerTopology(discoveredTopology);
         updateTopologyConfigId(topologyDescriptor.getTopologyConfigId());
-        log.debug("Persist new topologyConfigId {}, cluster id={}, role={}", topologyDescriptor.getTopologyConfigId(),
```

Comment:
Should we keep this around to keep it consistent with the logging in the `onClusterRoleChange` code path?